lib/services/ios-project-service.ts

```diff
@@ -230,6 +230,31 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		return exportFile;
 	}
 
+	/**
+	 * Exports .xcarchive for AppStore distribution.
+	 */
+	public async exportDevelopmentArchive(projectData: IProjectData, buildConfig: IBuildConfig, options: { archivePath: string, exportDir?: string, teamID?: string }): Promise<string> {
+		let platformData = this.getPlatformData(projectData);
+		let projectRoot = platformData.projectRoot;
+		let archivePath = options.archivePath;
+		let buildOutputPath = path.join(projectRoot, "build", "device");
+
+		// The xcodebuild exportPath expects directory and writes the <project-name>.ipa at that directory.
+		let exportPath = path.resolve(options.exportDir || buildOutputPath);
+		let exportFile = path.join(exportPath, projectData.projectName + ".ipa");
+
+		let args = ["-exportArchive",
+			"-archivePath", archivePath,
+			"-exportPath", exportPath,
+			"-exportOptionsPlist", platformData.configurationFilePath
+		];
+		await this.$childProcess.spawnFromEvent("xcodebuild", args, "exit",
+			{ stdio: buildConfig.buildOutputStdio || 'inherit', cwd: this.getPlatformData(projectData).projectRoot },
+			{ emitOptions: { eventName: constants.BUILD_OUTPUT_EVENT_NAME }, throwError: false });
```

Comment:
@yyosifov this has been a mistake - it should be true whenever we want to break the operation (i.e. the next operation has no meaning in case this one fails)